server/server.go

```diff
@@ -186,7 +188,10 @@ func (s *GrpcServer) NumToID(ctx context.Context, in *connect.Request[pbbmsrv.Nu
 	}
 
 	blockPbTimestamp := valueToTimestamp(response.KeyValues[0].Value)
-	return &connect.Response[pbbmsrv.BlockResp]{Msg: &pbbmsrv.BlockResp{Id: blockID, Num: blockNum, Time: blockPbTimestamp}}, nil
+	msg := &pbbmsrv.BlockResp{Id: blockID, Num: blockNum, Time: blockPbTimestamp}
+	sendMeteringEvent(ctx, proto.Size(msg), "sf.blockmeta.v2.Block/NumToID")
```

Comment:
The actual request ID/identifier is available through `req.Spec.Procedure`, would we want to use this instead?

Further question, would it make sense to have a middleware that automatically does the metering even sending?